dsalgo/stack.py

```diff
@@ -0,0 +1,125 @@
+class Node:
+	def __init__(self, value):
+		self.value = value
+		self.next = None
+
+
+class Stack_using_linked_list:
+	"""Implementation of Stack using Linked List"""
+
+	def __init__(self):
+		self.head = None
+		self.num_elements = 0
+
```

Comment:
I don't think that two blank lines should be added in function of same class. Can you please verify